pr-data.csv

```diff
@@ -6281,8 +6281,14 @@ https://github.com/oracle/oci-hdfs-connector,66d6b89415943fc20b0a591b09ee6d50bbe
 https://github.com/oracle/oci-hdfs-connector,66d6b89415943fc20b0a591b09ee6d50bbe7f9d0,hdfs-connector,com.oracle.bmc.hdfs.caching.CachingObjectStorageTest.testTwoGets_Cached_StrongConsistency_ifMatch_changed,ID,Opened,https://github.com/oracle/oci-hdfs-connector/pull/70,
 https://github.com/oracle/oci-java-sdk,2e9982f268420efd4010d8dc8370e9d9ade67ec2,bmc-objectstorage/bmc-objectstorage-extensions,com.oracle.bmc.objectstorage.transfer.ProgressTrackerTest.multipartUploadProgressTrackerFactory_simulateRealLifeScenario,NOD,,,
 https://github.com/oracle/oci-java-sdk,2e9982f268420efd4010d8dc8370e9d9ade67ec2,bmc-objectstorage/bmc-objectstorage-extensions,com.oracle.bmc.objectstorage.transfer.ProgressTrackerTest.singleUploadProgressTrackerFactory_resetProgressTracker,NDOD,,,
-https://github.com/oracle/tribuo,7d78a0ab15c2431d3ba935ea54c2aebbdba9c41d,Classification/DecisionTree,org.tribuo.classification.dtree.TestCART.testPureLeaf,ID,Opened,https://github.com/oracle/tribuo/pull/419,
-https://github.com/oracle/tribuo,7d78a0ab15c2431d3ba935ea54c2aebbdba9c41d,Classification/DecisionTree,org.tribuo.classification.dtree.TestClassificationEnsembles.load431ProtobufModel,ID,Opened,https://github.com/oracle/tribuo/pull/420,
+https://github.com/oracle/tribuo,7d78a0ab15c2431d3ba935ea54c2aebbdba9c41d,Classification/DecisionTree,org.tribuo.classification.dtree.TestCART.testPureLeaf,ID,InspiredAFix,https://github.com/oracle/tribuo/pull/421,
```

Comment:
Keep in `PR Link` 419 as the PR you opened, and in the Notes can put the PR 421 that fixed the case.